presentation/valid/ttml2-prstn-audio-in-body.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt xml:lang="en"
+    xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:tta="http://www.w3.org/ns/ttml#audio"
+    ttp:profile="ttml2-presentation"
+    xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="0" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <body>
+    <audio src="../resources/audio/ttml2-prstn-audio-clipbegin-end.mp3" type="audio/mpeg"/>
```

Comment:
```suggestion
    <audio src="../resources/audio/ttml2-prstn-audio-clipbegin-end.mp3" type="audio/mpeg"/>
    <div>
      <p>Early in. On time. Late Out.</p>
    </div>
```

Reply:
Addressed in f7edee7.

Reply:
Thanks @skynavga that now works in my implementation.